### ISSUE TITLE
Rule Change: Allow relative color syntax for <color>

### ISSUE THREAD

**Issue by amxmln** [eslint/css#235]
### What rule do you want to change?

css/no-invalid-properties

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```css
background-color: rgb(from currentColor r g b / 0.1)
```

### What does the rule currently do for this code?

This code currently gets flagged as incorrect, as `from` is not recognised as a valid property.

### What will the rule do after it's changed?

The rule should allow for relative color syntax.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

Please feel free to recategorise if I didn’t sort this into the right section, it just seemed to make the most sense. I’m surprised the issue wasn’t reported yet, however, I couldn’t find any references to it in other issues. If this is already known, I apologise.

**Comment by SwetaTanwar** [comment]
@snitin315 Are you still working on this? If not, can I take this up?

**Comment by snitin315** [comment]
@SwetaTanwar sure, all yours.

**Comment by nzakas** [comment]
Reopening because we haven't landed updated logic in the plugin yet. We need to add tests.

**Comment by snitin315** [comment]
Closing this issue as the reported error has been fixed, remaining syntaxes are being tracked in https://github.com/eslint/css/issues/246
